guides/private-app/src/main.ts

```diff
@@ -0,0 +1,28 @@
+import './style.css'
+import { initAuth } from './auth'
+import { createMap } from './map'
+import { setDefaultCredentials }from '@deck.gl/carto/typed'
+import { getAirports } from './api'
+
+document.querySelector<HTMLDivElement>('#app')!.innerHTML = `
```

Comment:
I would put this directly into index.html

Reply:
aggree, I think code will be simpler

Reply:
I prefer to keep as it's, because it's the default behaviour of `npm vite`. I mean, if you run npm vite@latest and select Vanilla and Typescript it will give you like this

Reply:
Fair enough